change/@fluentui-react-charting-1f77b865-250d-4cb3-bb5a-93fbdbf94a7c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think we can get past setting the min width and height prop if we don't touch the existing behavior if we make our min width requirement a bit more relaxed than the current max label length X (no of ticks - 1)